src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/viewmodels/IndexViewModel.kt

```diff
@@ -30,9 +30,19 @@ data class IndexViewModel(@Serialise("reportsJson") val reports: List<ReportRowV
                 customFieldKeys: List<String>,
                 showProjectDocs: Boolean,
                 canSetPinnedReports: Boolean,
-                reportDisplayNames: Map<String, String>?)
-            : this(reports, tags, pinnedReports, customFieldKeys, showProjectDocs,
-                    canSetPinnedReports, reportDisplayNames, DefaultViewModel(context, breadcrumb))
+                reportDisplayNames: Map<String, String>?) : this(reports, tags, pinnedReports, customFieldKeys,
+            showProjectDocs, canSetPinnedReports, reportDisplayNames, DefaultViewModel(context, breadcrumb))
+
+    constructor(context: ActionContext,
```

Comment:
Why does this need a second constructor? Nothing should be using the signature without `isRunner` should it? (Connected to Mark's remark about `isRunner` not needing to be nullable.)